qabelbox/src/main/res/values/strings.xml

```diff
@@ -209,9 +209,57 @@
 
     <string name="licence.header.intro">Qabel is Published under the QAPL:</string>
     <string name="about.header.qabel.company.qtxt">QABEL</string>
-    <string name="about.header.qabel.show.btn">Show QAPL</string>
+    <string name="about.header.qabel.show.btn">Show QaPL</string>
     <string name="about.header.qabel.thanks.header.txt"> bedankt sich bei:</string>
-    <string name="about.header.qabel.thanks.content.txt">praemandatum GmbH\nden Unterstützern unserer Indiegogo Kampagne\nHinz\nund\nKunz</string>
+    <string name="about.header.qabel.thanks.content.txt">Alexander Hartmann
```

Comment:
This text must begin with `Qabel bedankt sich bei:` plus two newlines.

KEEP IN MIND that the text `Qabel` at the beginning is shown as the Qabel logo.